src/render.rs

```diff
@@ -41,18 +45,57 @@ pub fn emit_commandline(
         format!("oci://{image}")
     };
 
-    let mut cli = [
-        "kubecfg",
-        "show",
-        "--alpha",
-        "--reorder=server",
-        &entrypoint,
-        "--overlay-code-file",
-        &format!("appInstance_={overlay_file}"),
-    ]
-    .iter()
-    .map(|s| s.to_string())
-    .collect::<Vec<String>>();
+    let mut cli: Vec<String> = vec![];
+
+    if is_local {
+        let overlay_path = std::fs::canonicalize(overlay_file).unwrap();
+        let overlay_file_name = std::path::PathBuf::from(overlay_path.file_name().unwrap());
+        let user_home = home_dir().expect("unable to retrieve home directory");
+
+        cli = vec![
+            "docker",
+            "run",
+            "--rm",
+            "-v",
+            &format!("{}/.kube/config:/.kube/config", user_home.display()),
+            "-v",
+            &format!(
+                "{}:/overlay/{}",
+                overlay_path.display(),
+                overlay_file_name.display()
+            ),
+            "-v",
+            &format!(
+                "{}/.docker/config.json:/.docker/config.json:ro",
+                user_home.display()
```

Comment:
we should honour the `DOCKER_CONFIG` env var and default to `format!("{}/.docker", user_home.display()` if the DOCKER_CONFIG env var is not set